benches/src/db_lookup_times_utils/full_block_table.rs

```diff
@@ -0,0 +1,60 @@
+use fuel_core::database::database_description::DatabaseDescription;
+use fuel_core_storage::kv_store::StorageColumn;
+use fuel_core_types::fuel_types::BlockHeight;
+
+#[repr(u32)]
+#[derive(
+    Copy,
+    Clone,
+    Debug,
+    strum_macros::EnumCount,
+    strum_macros::IntoStaticStr,
+    PartialEq,
+    Eq,
+    enum_iterator::Sequence,
+    Hash,
+    num_enum::TryFromPrimitive,
+)]
+pub enum BenchDbColumn {
+    /// See [`Transactions`](crate::tables::Transactions)
+    Transactions = 6,
+    /// See [`FuelBlocks`](crate::tables::FuelBlocks)
+    FuelBlocks = 7,
+    FullFuelBlocks = 10902,
+    Metadata = 10903,
```

Comment:
Maybe 0, 1, 2, 3 will be simpler=D

Reply:
addressed in [d09740c](https://github.com/FuelLabs/fuel-core/pull/2159/commits/d09740c1d4d955042493b84ad0db963bca3cbf74)